src/App.tsx

```diff
@@ -1,17 +1,128 @@
-import React from 'react';
+import React, { useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 import './App.css';
 
-import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './react-app-env';
 
 const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([...todosFromServer]);
+
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [completed, setCompleted] = useState(false);
+
+  const [titleIsEmpty, setTitleIsEmpty] = useState(false);
+  const [userIdIsEmpty, setUserIdIsEmpty] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent): void => {
+    event.preventDefault();
+
+    if (title === '') {
+      setTitleIsEmpty(true);
+    }
+
+    if (userId === 0) {
+      setUserIdIsEmpty(true);
+    }
+
+    if (title && userId) {
+      setTodos([...todos, {
+        userId,
+        id: todos.length + 1,
+        title,
+        completed,
+      }]);
```

Comment:
Will be better to use function inside setTodos, if you want to change the state depending on the previous value